### PR TITLE
support rollup function in druid 0.9.2

### PR DESCRIPTION
**Pull request by GavinGavinNo1**
RT , support ability to disable rollup released in druid V0.9.2.

### CODE REVIEW

**Comment by GavinGavinNo1** [comment]
@gianm Could you have a check on this? Tests aren't passed because they were not before.

**Comment by gianm** [comment]
Hey @GavinGavinNo1, thanks for the contribution! Reviewing now.

Could you also please fill out our CLA at: http://druid.io/community/cla.html (it's the same CLA as Druid). Thanks!

**Comment by gianm** [comment]
The tests are a little sketchy when running on Travis unfortunately… but they should pass eventually. I'll try running them again.

File: core/src/main/scala/com/metamx/tranquility/druid/DruidRollup.scala [hunk 1]
```diff
@@ -35,7 +35,8 @@ import scala.collection.JavaConverters._
 class DruidRollup(
   val dimensions: DruidDimensions,
   val aggregators: IndexedSeq[AggregatorFactory],
-  val indexGranularity: QueryGranularity
+  val indexGranularity: QueryGranularity,
+  val isRollup: Boolean
```
Comment (gianm) [inline core/src/main/scala/com/metamx/tranquility/druid/DruidRollup.scala]: Can you give this a default value of `true` for backwards compatibility with existing source? (people won't have the extra arg defined)

File: core/src/test/scala/com/metamx/tranquility/test/DruidBeamTest.scala [hunk 2]
```diff
@@ -56,6 +54,39 @@ import _root_.scala.collection.JavaConverters._
 
 class DruidBeamTest extends FunSuite with Matchers
 {
+  private def emptyEmitter(): ServiceEmitter = new ServiceEmitter(
+    "service", "host", new Emitter
```
Comment (gianm) [inline core/src/test/scala/com/metamx/tranquility/test/DruidBeamTest.scala]: You can use `NoopEmitter` for this dummy emitter.

**Comment by GavinGavinNo1** [comment]
@gianm I have already updated it.

**Comment by gianm** [comment]
Thanks @GavinGavinNo1, 👍 on this change. Could you please fill out the CLA (http://druid.io/community/cla.html) so we can merge it?

**Comment by gianm** [comment]
Hey @GavinGavinNo1, for individuals it's simpler for us (and you) if you fill out the online form; would you mind doing that?

**Comment by GavinGavinNo1** [comment]
@gianm Not at all.  I filled out the online form just now, please check it. Thanks.

**Comment by gianm** [comment]
thanks @GavinGavinNo1, got it!

**Comment by fjy** [comment]
👍

**Pull request merged**